### PR TITLE
Setup travis

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Set up the CI with travis.

> Migration to original repositery of forked PR: https://github.com/vscode-org-mode/vscode-org-mode-fork/pull/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # VS Code Org Mode
+[![Version](https://vsmarketplacebadge.apphb.com/version/tootone.org-mode.svg)](https://marketplace.visualstudio.com/items?itemName=tootone.org-mode)
```
Comment (sipi) [inline README.md]: duplicate

File: README.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+[![Installs](https://vsmarketplacebadge.apphb.com/installs/tootone.org-mode.svg)](https://marketplace.visualstudio.com/items?itemName=tootone.org-mode)
```
Comment (sipi) [inline README.md]: duplicate

File: README.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
+[![Ratings](https://vsmarketplacebadge.apphb.com/rating/tootone.org-mode.svg)](https://marketplace.visualstudio.com/items?itemName=tootone.org-mode)
```
Comment (sipi) [inline README.md]: duplicate

File: README.md [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
+[![Travis](https://img.shields.io/travis/com/vscode-org-mode/vscode-org-mode.svg)](https://travis-ci.com/vscode-org-mode/vscode-org-mode)
```
Comment (sipi) [inline README.md]: to move before header or move those which are before after

File: README.md [hunk 5]
```diff
@@ -1,4 +1,9 @@
...
+[![codecov](https://codecov.io/gh/vscode-org-mode/vscode-org-mode/branch/master/graph/badge.svg)](https://codecov.io/gh/vscode-org-mode/vscode-org-mode)
```
Comment (sipi) [inline README.md]: same as previous line

File: README.md (hunk 1 above)
Comment (rpeshkov) [reply README.md]: @sipi, I have the same problem in my PRs. Your commit 8e80df9 looks like to be in each latest PR :)

Comment (AdrieanKhisbe) [reply README.md]: @sipi @rpeshkov I solved it by doing an interactive rebase and getting ride of the commit

**Comment by AdrieanKhisbe** [comment]
@sipi With the interactive rebase it should be good now =)

**Review by sipi — APPROVED** [review]

**Review by rpeshkov — APPROVED** [review]

**Comment by AdrieanKhisbe** [comment]
Two approval. Squashing it :)

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+install:
+  - npm install
+  - npm run vscode:prepublish
+
+script:
+  - npm test
+
+after_success:
+  - npm run publish-coverage
```
Comment (sipi) [inline .travis.yml:22]: Hi,
you introduced "npm run publish-coverage" but there is not "publish-coverage" script in package.json !

Comment (AdrieanKhisbe) [reply .travis.yml:22]: oooopps

Comment (sipi) [reply .travis.yml:22]: :), I tried to replace this line by this other one in a "test/travis" branch ```  - bash <(curl -s https://codecov.io/bash)```, but I still have nothing in https://codecov.io/gh/vscode-org-mode/vscode-org-mode even after a Travis build…

**Pull request merged**